quipucords/api/source/view.py

```diff
@@ -116,6 +117,26 @@ def create(self, request, *args, **kwargs):
 
         # Create expanded host cred JSON
         expand_credential(json_source)
+
+        # check to see if a connection scan was requested
+        # through query parameter
+        scan = request.query_params.get('scan', False)
+        # If the scan was requested, create a connection scan
+        if scan:
+            # Grab the last source that was created
+            scan_source = self.queryset.latest('id')
```

Comment:
I would think that the response object has the `id` and you shouldn't need to query for it.